src/cli/simple/command/mod.rs

```diff
@@ -1069,3 +1069,33 @@ impl ContractCall<'_> {
         Ok(())
     }
 }
+
+pub async fn add_relayer(
+    context: Context,
+    account_id: String,
+    deposit: f64,
+    full_access_pub_key: near_crypto::PublicKey,
+    function_call_pub_key: near_crypto::PublicKey,
+) -> anyhow::Result<()> {
+    let mut client = context.client.near();
+    let engine_account_id = format!("relay.{account_id}");
+
+    client.engine_account_id = engine_account_id.parse()?;
```

Comment:
I guess it's not needed. Try not to do it.

Reply:
Then there will be an error like `Account already exists xxx.c.aurora` if we will keep the engine parameter the same.
On `near_broadcast_tx_from` `signer_id` and `receiver_id` will be the same value.

Reply:
Yes. That's right. Ok